### PR TITLE
Player can now rotate remaining tile if it is its turn

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
Depends on #27 

Capture vidéo du 08-03-2023 13_45_33

### CODE REVIEW

File: mobile/src/board/BoardPages.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { boardRepository } from "./BoardRepository";
+
+export function GetById() {
+  const { id } = useParams();
+  const [board, error] = useBoard(id!);
+
+  const rotateRemainingTile = async () => {
+    boardRepository.rotateRemainingTile(id!);
+  };
+
```
Comment (djhi) [inline mobile/src/board/BoardPages.tsx:14]: Why is this here instead of in `BoardView` directly ? Besides, FYI, the convention for callbacks is to name the prop `onEvent` and the handler `handleEvent`. Here that would be `onRotateRemainingTile` and `handleRemainingTile`

**Review by djhi — CHANGES_REQUESTED** [review]

File: mobile/src/board/BoardPages.tsx (hunk 1 above)
Comment (djhi) [reply mobile/src/board/BoardPages.tsx:14]: If you wanted to keep the repository logic outside `BoardView`, please just rename the prop and handler

Comment (jonathan-marmelab) [reply mobile/src/board/BoardPages.tsx:14]: I wanted to keep the repository logic outside the `BoardView` , I will rename the props and handlers

File: mobile/src/board/BoardPages.tsx [hunk 2]
```diff
@@ -8,15 +8,17 @@ export function GetById() {
   const { id } = useParams();
   const [board, error] = useBoard(id!);
 
-  const rotateRemainingTile = async () => {
+  const onRotateRemainingTile = async () => {
```
Comment (djhi) [inline mobile/src/board/BoardPages.tsx:11]: Why is it `async` if you don't `await`?

**Review by djhi — APPROVED** [review]
